src/Documentation/DocumentationGenerator/DocumentationWriter.cs

```diff
@@ -149,10 +149,13 @@ public async Task WriteOutput(QsNamespace ns, DocComment docComment)
                 // DocFX metadata
                 ["uid"] = name,
                 ["title"] = title,
+                ["author"] = "msquantumdocs",
+                ["description"] = docComment.Description,
```

Comment:
This looks weird, but in docs.ms parlance, a description is a short one-sentence summary, but in API documentation parlance, that same thing is the summary, and the description can be much longer.
```suggestion
                ["description"] = docComment.Summary,
```